openwisp_radius/tests/test_api/test_freeradius_api.py

```diff
@@ -298,6 +298,18 @@ def test_postauth_400(self):
         self.assertEqual(RadiusPostAuth.objects.all().count(), 0)
         self.assertEqual(response.status_code, 400)
 
+    def test_postauth_called_station_id_max_length_50_exceed_400(self):
+        params = {'called_station_id': 'C0-4A-00-EE-D1-0D:' + 'A' * 50}
+        params = self._get_postauth_params(**params)
+        response = self.client.post(
+            reverse('radius:postauth'), params, HTTP_AUTHORIZATION=self.auth_header
+        )
+        self.assertEqual(response.status_code, 400)
+        self.assertEqual(
+            response.data['called_station_id'][0],
+            'Ensure this field has no more than 50 characters.',
+        )
```

Comment:
This test looks good. Can you also test the `calling_station_id` field?

Reply:
added `calling_station_id` field in the params with its `length > 50`
added assertion for `calling_station_id` field has more than 50 characters as well